src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -3,6 +3,74 @@
 <html>
 	<head>
 		<script>
+			async function gitpodMetricsAddCounter(metricsName, labels, value) {
```

Comment:
suggestion (non-blocking): Can we use trailing semicolons for the whole function? The rest of the workbench seems to use it.